### PR TITLE
Added metrics for bulk batch saving

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

We did not have metrics enabled for the "bulk batch saving" feature. Either through the API or admin. This PR adds metrics for that feature.

### CODE REVIEW

File: tests/app/celery/test_ftp_update_tasks.py [hunk 1]
```diff
@@ -159,13 +159,10 @@ def test_update_letter_notifications_statuses_persisted(notify_api, mocker, samp
...
     assert failed_letter.billable_units == 2
     assert failed_letter.updated_at
-    assert (
-        "DVLA response file: {filename} has failed letters with notification.reference {failures}".format(
-            filename="NOTIFY-20170823160812-RSP.TXT",
-            failures=[format(failed_letter.reference)],
-        )
-        in str(e)
-    )
+    assert "DVLA response file: {filename} has failed letters with notification.reference {failures}".format(
```
Comment (jzbahrai) [inline tests/app/celery/test_ftp_update_tasks.py:162]: black reformatted this 🤷🏽

File: tests/app/dao/test_permissions_dao.py [hunk 2]
```diff
@@ -5,21 +5,18 @@
...
-                "send_texts",
-                "send_emails",
-                "send_letters",
-                "manage_api_keys",
-                "view_activity",
-            ]
-        )
-        == sorted([i.permission for i in permissions])
-    )
+    assert sorted(
```
Comment (jzbahrai) [inline tests/app/dao/test_permissions_dao.py:8]: black reformatted this 🤷🏽

File: tests/app/test_config.py [hunk 3]
```diff
@@ -59,29 +59,26 @@ def test_queue_names_all_queues_correct():
...
-                # QueueNames.CREATE_LETTERS_PDF,
-                QueueNames.CALLBACKS,
-                # QueueNames.LETTERS,
-                QueueNames.DELIVERY_RECEIPTS,
-            ]
-        )
-        == set(queues)
-    )
+    assert set(
+        [
```
Comment (jzbahrai) [inline tests/app/test_config.py:63]: black reformatted this

**Comment by jimleroyer** [comment]
I am a bit mixed on these changes. What's the goal of these new metrics?

**Comment by jzbahrai** [comment]
@jimleroyer We have metrics around batch insertion if it comes through the API. We should have similar metrics for Batch Insertion if it comes through the bulk API or the bulk admin call. This way we know things have been sent and processed for the bulk call as well.

**Review by jimleroyer — APPROVED** [review]
LGTM!

**Pull request merged**